packages/selleo-design-core/src/Button.tsx

```diff
@@ -1,61 +1,29 @@
 import { h } from "preact";
+import classNames from "classnames";
 
-export function PrimaryButtonSmall() {
-  return <button class="inline-flex py-1 px-2 gap-0.5 transition duration-300 justify-center text-xs text-white font-bold leading-7 tracking-normal bg-brand-primary-500 hover:bg-brand-primary-600 focus-visible:bg-brand-primary-600 focus-visible:outline focus-visible:outline-offset-1 focus-visible:outline-1 focus-visible:outline-brand-primary-600 rounded disabled:bg-brand-primary-500 disabled:opacity-30">Text</button>;
-}
-
-export function PrimaryButton() {
-  return <button class="inline-flex py-1 px-3 gap-1 transition duration-300 justify-center text-sm text-white font-bold leading-7 tracking-normal bg-brand-primary-500 hover:bg-brand-primary-600 focus-visible:bg-brand-primary-600 focus-visible:outline focus-visible:outline-1 focus-visible:outline-brand-primary-600 focus-visible:outline-offset-2 rounded disabled:bg-brand-primary-500 disabled:opacity-30">Text</button>;
-}
+export function Button(
+  { primaryOutlined, link, small, ...rest }: { primaryOutlined?: boolean, link?: boolean, small?: boolean } & h.JSX.HTMLAttributes<HTMLButtonElement>
```

Comment:
let's have props like `variant = 'primary'  | 'primaryOutlined' etc`
and `size='small' | 'normal'` with default values being primary and normal